presto-parser/src/main/java/com/facebook/presto/sql/ExpressionFormatter.java

```diff
@@ -95,18 +95,22 @@
 import static com.facebook.presto.sql.SqlFormatter.formatSql;
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.collect.Iterables.getOnlyElement;
-import static java.lang.String.format;
```

Comment:
Can we rename this commit to "Add TemplateExpressionFormatter" or something like that?

Reply:
Renamed commit as suggested